exec/cnex/lib/io.c

```diff
@@ -22,6 +22,31 @@
 #include "number.h"
 #include "stack.h"
 #include "nstring.h"
+#include "object.h"
+
+
+void object_releaseFileObject(Object *o)
+{
+    if (o != NULL) {
+        assert(o->refcount > 0);
+        o->refcount--;
+
+        if (o->refcount <= 0) {
+            if (o->ptr != NULL) {
+                fclose(o->ptr);
```

Comment:
You will probably have to cast this to a `FILE*` for at least some compiler.

Reply:
Fixed.